Formula/tre-command.rb

```diff
@@ -0,0 +1,19 @@
+class TreCommand < Formula
+  desc "Tree command, improved"
+  homepage "https://github.com/dduan/tre"
+  url "https://github.com/dduan/tre/archive/v0.3.1.tar.gz"
+  sha256 "3d7a7784ed85dd5301f350a3d05eca839f24846997eb0a44b749467f0f4dd032"
+  head "https://github.com/dduan/tre.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked",
+                               "--root", prefix,
+                               "--path", "."
+  end
+
+  test do
+    system "#{bin}/tre", testpath
```

Comment:
Can we test some output here?

Reply:
I like the idea. But it's hard to think of specific things. The app has its own integration test. I think that's not appropriate to run here. The test as is verifies that the binary exists and exits with a success code. ripgrep does the same thing, for example: https://github.com/Homebrew/homebrew-core/blob/master/Formula/ripgrep.rb#L35-L38

Reply:
Since we usually don't retroactively apply test improvements referring to other formulae for tests isn't always reliable. But if it's a `tree` alternative making a file and checking it exists in the output should be doable right?

Reply:
I think that test should be part of the tool itself and not as part of its packaging. But I'll give it a try here.

Reply:
It checks if a user can perform a common action with the tool without failing, it fits the description of a "good test" in https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
updated the test